docs/modules/ROOT/pages/guides/guides-local-multi-node.adoc

```diff
@@ -0,0 +1,106 @@
+* Example: `k3d cluster create three-node-cluster --agents 3 --registry-create cluster-registry`
+4. Point `kubectl` to your new cluster
+* `kubectl config use-context <cluster-name>`
+* Example: `kubectl config use-context three-node-cluster`
+
+=== Assigning aiSSEMBLE(TM) Components to Particular Nodes or Node Types
+The recommended approach to assigning pods to nodes is using Kubernetes
+https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity[Node
+Affinity,role=external,window=_blank], which allows you to define node selection logic. Within your aiSSEMBLE project,
+Helm charts which define Kubernetes deployments for the various components of your project can be found within the
+
+`<project-name>-deploy/src/main/resources/apps` 
+
+directory. To define Node Affinity rules for a particular component, navigate to the 
+
+`<project-name>-deploy/src/main/resources/apps/<component-name>/templates/deployment.yaml` 
+
+template and add an `affinity` section directly under the `spec.template.spec` section. Detailed information regarding
+how to configure this section to contain hard/required and soft/preferred affinity rules can be found
+https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity[here,role=external,window=_blank].
+To run a simple example, you can use the following configuration requiring your component to be executed on a node with
+the label key/value pair `"label-key":"label-value"`:
+
+[source,yaml]
+----
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: label-key
+                operator: In
+                values:
+                - label-value
+----
+
+Once you've configured your rules, you need to label your node(s) accordingly. Consult the
+https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes/#add-a-label-to-a-node[Kubernetes
+pod documentation,role=external,window=_blank] for information on how to do so. Continuing with our simple example,
+perform the following steps:
+
+1. Retrieve node names by running: `kubectl get nodes`
+2. Choose one of the nodes with `agent` in its name
+3. Add label to node: `kubectl label nodes <node-name> node-label-key=node-label-value`
+* Example: `kubectl label nodes k3d-three-node-cluster-agent-0 label-key=label-value`
+
+Now that you've configured your Node Affinity rules and labeled your node(s), you can run `tilt up` from the root
+directory of your aiSSEMBLE project to deploy your project to your multi-node cluster. Finally, to verify that your
+rules are being followed, you can run the following command to see which pods are running on which nodes:
+
+`kubectl get pods -o wide --field-selector spec.nodeName=<node-name>`
+
+=== Configuring Horizontal Pod Autoscaling 
```

Comment:
A: Formatting - Similar wrapping issues here around the code examples but far less egregious.

Reply:
✅ UPDATED